starknet-foundry/crates/forge/Cargo.toml

```diff
@@ -15,15 +15,15 @@ tempfile.workspace = true
 rayon.workspace = true
 test-collector = { path = "../test-collector" }
 cheatable-starknet = { path = "../cheatable-starknet" }
-cairo-lang-test-runner = { path = "../../../cairo/crates/cairo-lang-test-runner", version = "2.0.1" }
-cairo-lang-runner = { path = "../../../cairo/crates/cairo-lang-runner", version = "2.0.1" }
-cairo-lang-sierra-to-casm = { path = "../../../cairo/crates/cairo-lang-sierra-to-casm", version = "2.0.1" }
-cairo-lang-sierra = { path = "../../../cairo/crates/cairo-lang-sierra", version = "2.0.1" }
-cairo-lang-utils = { path = "../../../cairo/crates/cairo-lang-utils", version = "2.0.1" }
-cairo-lang-casm = { path = "../../../cairo/crates/cairo-lang-casm", version = "2.0.1" }
-cairo-lang-starknet = { path = "../../../cairo/crates/cairo-lang-starknet", version = "2.0.1" }
-cairo-lang-compiler = { path = "../../../cairo/crates/cairo-lang-compiler", version = "2.0.1" }
-cairo-lang-filesystem = { path = "../../../cairo/crates/cairo-lang-filesystem", version = "2.0.1" }
+cairo-lang-test-runner = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-runner = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-sierra-to-casm = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-sierra = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-compiler = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
+cairo-lang-filesystem = { git = "https://github.com/starkware-libs/cairo", rev = "f9e7b41" }
```

Comment:
It has to be this way for now, since 2.0.2 doesn't include changes that made some much needed functions public. We cannot use 2.1.0-rc0 since !(2.1.0-rc0 >= 2.0.0) and blockifier uses 2.0.0.

This commit points to the main branch merged with 2.0.2

Reply:
If we merge with this, and issue to use deps directly from crates.io should be created.

Reply:
I think it is kinda obvious, but will link this comment in the issue

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/68#issuecomment-1643615388